serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -462,26 +463,39 @@ curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d
 [[proc-deploy-sw-application-quarkus-cli]]
 === Deploying your Serverless Workflow application using Quarkus CLI
 
-The link:{quarkus_cli_url}[Quarkus CLI] can be used to abstract all of these complexity mentioned before and deploy it automatically after the build completes. To instruct Quarkus to perform the deployment automatically for you, you need to set a few system properties, which are:
+You can use the link:{quarkus_cli_url}[Quarkus CLI] to abstract all of the complexities mentioned in the previous sections and deploy your Serverless Workflow application once the build completes. 
 
-.System Properties
+
+.Prerequisites
+* Serverless Workflow application is installed.
+* Quarkus CLI is installed. For more information, see link:{quarkus_cli_url}[Building Quarkus Apps with Quarkus command line interface (CLI)].
+
+.Procedure
+. To deploy your Serverless Workflow application using Quarkus CLI, set the following system properties in `application.properties` file:
++
+--
+.Required system properties
 [source,properties]
 ----
 quarkus.knative.name=greeting-quarkus-cli <1>
 quarkus.kubernetes.deploy=true <2>
 quarkus.container-image.push=false <3>
 ----
-<1> - The Knative resource name.
-<2> - Whether or not install deploy the on the active kubernetes cluster.
-<3> - Force to not push images when using Minikube's remove Docker daemon to avoid image validation.
+
+<1> - Property for the Knative resource name
+<2> - Property to set whether or not deploy on an active Kubernetes cluster
+<3> - Property to whether or not push images when using Minikube's remove Docker daemon to avoid image validation
 
 [IMPORTANT]
 ====
-This functionality only works with Quarkus 2.10.2.Final or newer, for more information please visit this
+This functionality works with Quarkus {quarkus_version} or later. For more information, see
```

Comment:
The specific bug that was preventing us to was fixed on 2.10.2.Final, thus it needs to be hardcoded.